starport/services/serve/rly/relayer.go

```diff
@@ -0,0 +1,82 @@
+package rly
+
+import "fmt"
+
+type GlobalConfig struct {
+	Timeout       string `yaml:"timeout" json:"timeout"`
+	LiteCacheSize int    `yaml:"lite-cache-size" json:"lite-cache-size"`
+}
+
+type Chains []*Chain
+
+type Chain struct {
+	Key            string  `yaml:"key" json:"key"`
+	ChainID        string  `yaml:"chain-id" json:"chain-id"`
+	RPCAddr        string  `yaml:"rpc-addr" json:"rpc-addr"`
+	AccountPrefix  string  `yaml:"account-prefix" json:"account-prefix"`
+	Gas            uint64  `yaml:"gas,omitempty" json:"gas,omitempty"`
+	GasAdjustment  float64 `yaml:"gas-adjustment,omitempty" json:"gas-adjustment,omitempty"`
+	GasPrices      string  `yaml:"gas-prices,omitempty" json:"gas-prices,omitempty"`
+	DefaultDenom   string  `yaml:"default-denom,omitempty" json:"default-denom,omitempty"`
+	Memo           string  `yaml:"memo,omitempty" json:"memo,omitempty"`
+	TrustingPeriod string  `yaml:"trusting-period" json:"trusting-period"`
+}
+
+type Paths map[string]*Path
+
+type Path struct {
```

Comment:
Same as above, what that path is used for in IBC?

Reply:
added an explanation for this in 9f4ad8c